linux_os/guide/system/crypto_policies/group.yml

```diff
@@ -0,0 +1,5 @@
+documentation_complete: true
```

Comment:
@matejak this file needs to be removed from this PR, but after that, I think that it is good to merge.